src/routes/boot.ts

```diff
@@ -394,6 +394,7 @@ const COMPANION_QUERY = parse(`query Post($url: String) {
           author {
             id
           }
+          downvoted
```

Comment:
Not sure, but I thought we also had a specific companion boot test that should be affected.

Reply:
Please validate it before merging :)

Reply:
@rebelchris this one 92972601b44b0444789e6d45d00d305be11b1256?

Reply:
Ah yes, sorry thought that was the regular boot (sometimes hard to see in the PR screen 😅)